Source/macOS/OIDRedirectHTTPHandler.h

```diff
@@ -45,17 +45,30 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (instancetype)initWithSuccessURL:(nullable NSURL *)successURL;
 
+/*! @brief Starts listening on the loopback interface on a specified port, and returns a URL
+        with the base address. Use the returned redirect URI to build a @c OIDExternalUserAgentRequest,
+        and once you initiate the request, set the resulting @c OIDExternalUserAgentSession to
+        @c currentAuthorizationFlow so the response can be handled.
+    @param returnError The error if an error occurred while starting the local HTTP server.
+    @param port The port that the user would like to manually specify or 0 for a random port
```

Comment:
"the user would like to" wording is redundant, suggest changing `The port that the user would like to manually specify`

To something more like `The manually specified port, or 0 for`